app/js/preloads/downloads.js

```diff
@@ -97,6 +97,9 @@ const addDownload = (download, id) => {
     const title = document.createElement("div")
     title.textContent = download.name
     title.className = "title"
+    title.addEventListener("click", () => {
+        ipcRenderer.send("open-download", download.file)
```

Comment:
If we're changed the click region from just the filename to something else I think it would make more sense to have the entire block work as a file open action, with the same hover title you set on the notification page. This would make it work the same as on that page as well, to keep it consistent.

Reply:
I'm not against this, but it might be a bit confusing: there is already the URL link and the trash button in the download div, where a click means something completely different.